packages/docs/content/14-troubleshooting.md

```diff
@@ -190,3 +190,147 @@ Installation of the gateway cluster may run into errors or might get stuck in th
 * [Kubernetes Dashboard](/integrations/kubernetes-dashboard/#troubleshooting)
 * [CSI](/integrations/csi/#troubleshooting)
 
+# Palette Tenant Cluster NameSpaces with Pods
+
+|PALETTE NAMESPACES | PODS  |
+|------|-------|                                                        
+|capi-webhook-system    |capi-controller-manager-< UUID>|                       
+||capi-kubeadm-bootstrap-controller-manager-< UUID >|      
+||capi-kubeadm-control-plane-controller-manager-< UUID >| 
+|capi-webhook-system|capv-controller-manager-< UUID >|                        
+|cert-manager|cert-manager-< UUID >|
+||cert-manager-cainjector-< UUID> |                        
+||cert-manager-webhook-< UUID > |                         
+|cluster-< UUID >  | capi-controller-manager-< UUID >
+||capi-kubeadm-bootstrap-controller-manager-< UUID >        
+||capi-kubeadm-control-plane-controller-manager-< UUID >  
+||capv-controller-manager-< UUID >                        
+||cluster-management-agent-< UUID >
+|cluster-< UUID > |metrics-server-< UUID > |                   
+| |palette-controller-manager-< UUID >|                    
+|kube-system |calico-kube-controllers-< UUID >                         
+| |calico-node-< UUID >
+| |calico-node-< UUID >                                              
+| |coredns-< UUID >
+| |coredns-< UUID > |                                         
+| |etcd-vmdynamictest-cp-< UUID >|                                     
+| |kube-apiserver-vmdynamictest-cp-< UUID > |                          
+| |kube-controller-manager-vmdynamictest-cp-< UUID > |                 
+| |kube-proxy-< UUID >  |                                                
+| |kube-proxy-< UUID >  |                                               
+| |kube-scheduler-vmdynamictest-cp-< UUID >  |                          
+| |kube-vip-vmdynamictest-cp-< UUID >  |                               
+|reach-system  |                 reach-controller-manager-< UUID > |    
+                   
+
+
+# Palette PCG NameSpaces with Pods
+
```

Comment:
Add "Here is a mapping of the pods and namespaces." before presenting the table.